docs/platforms/Swift-Package-Manager.md

```diff
@@ -0,0 +1,645 @@
+-                // .process("PrivacyInfo.xcprivacy"),
++                .process("PrivacyInfo.xcprivacy"),
+
+                // If you have other resources that need to be bundled with your plugin, refer to
+                // the following instructions to add them:
+                // https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package
+            ],
+```
+6. Move any resource files from `ios/Assets` to `Sources/plugin_name_ios` (or a subdirectory). Then add them to your Package.swift if applicable. See https://developer.apple.com/documentation/xcode/bundling-resources-with-a-swift-package for more instructions.
+7. Move any public headers from `ios/Classes` to `Sources/plugin_name_ios/include/plugin_name_ios`
+    * If you're unsure what headers are public, check your `podspec` for `public_header_files`. If not found, that means all of your headers were public. You should consider whether or not you want all of your headers to be public.
+    * The `pluginClass` defined in your pubspec.yaml must be public and within this directory.
+8. Handling modulemap (skip this step if not using a custom modulemap)
+
+    If you're using a modulemap for CocoaPods to create a Test submodule, consider removing it for Swift Package Manager. Note that this will make all public headers available via the module.
+
+    To remove it for Swift Package Manager but keep it for CocoaPods, exclude the modulemap and umbrella header in the plugin's Package.swift. The example below assumes they are located within the `Sources/plugin_name_ios` directory.
+
+    ```diff
+            .target(
+                name: "plugin_name_ios",
+                dependencies: [],
+    +           exclude: ["cocoapods_plugin_name_ios.modulemap", "plugin_name_ios-umbrella.h"],
+    ```
+
+    If you want to keep your unit tests compatible with both CocoaPods and Swift Package Manager, you can try the following:
+    ```diff
+    @import plugin_name_ios;
+    - @import plugin_name_ios.Test;
+    + #if __has_include(<plugin_name_ios/plugin_name_ios-umbrella.h>)
+    +   @import plugin_name_ios.Test;
+    + #endif
+    ```
+
+    If you would like to continue using a custom modulemap, please refer to [Swift Package Manager's documentation](https://github.com/apple/swift-package-manager/blob/main/Documentation/Usage.md#creating-c-language-targets).
+
+9. Move all remaining files from `ios/Classes` to `Sources/plugin_name_ios`
+10. `ios/Assets`, `ios/Resources`, `ios/Classes` should now be empty and can be deleted
+11. If your header files were previously within the same directory as your implementation files, you may need to change your import statements.
+
+   For example, if the following changes were made:
+     * `ios/Classes/PublicHeaderFile.h` --> `Sources/plugin_name_ios/include/plugin_name_ios/PublicHeaderFile.h`
+     * `ios/Classes/ImplementationFile.m` --> `Sources/plugin_name_ios/ImplementationFile.m`
```

Comment:
```suggestion
For example, if the following changes were made:
  * `ios/Classes/PublicHeaderFile.h` --> `Sources/plugin_name_ios/include/plugin_name_ios/PublicHeaderFile.h`
  * `ios/Classes/ImplementationFile.m` --> `Sources/plugin_name_ios/ImplementationFile.m`
```

Reply:
Good catch on this one. This is part of step 11, it looks like we need _more_ spaces to make GitHub markdown happy